sr_utilities/scripts/sr_utilities/hand_finder.py

```diff
@@ -141,6 +141,18 @@ def wait_for_hand_params(self, timeout_in_secs):
             self._hand_h = True
             self._hand_h_parameters = rospy.get_param("/fh_hand")
 
+    def is_type_hand_e(self):
+        return self._hand_e
+
+    def get_hand_e_parameters(self):
```

Comment:
Why not using the property decorator to actual make getters and setters?

Reply:
as this is not yet our standard so this has a better readability when reused elsewhere in my opinion. If this is not ok for you, are you happy with me changing this to:

@property
def hand_e_parameters(self):
        return self._hand_parameters
        
@property
def hand_h_parameters(self):
        return self._hand_h_parameters
        
?

Reply:
I'd say its a matter of standard to keep it consistent across all codes (either decorators or methods set/get, or even something else) or no? I'll add only getters as, this is supposed to be initialized from the parameter server.

Reply:
Even if it's set from the parameter server, you will do `self.param = blabla`, which would call the setter  ^^. And the difference between decorator and simple getter and setter runs much deeper than "standard"

Reply:
So we merge?